styles.css

```diff
@@ -366,3 +359,161 @@ textarea {
   line-height: 20px;
   color: #172b4d;
 }
+
+/* Responsive design starts here */
+
+@media all and (min-width: 1000px) {
+  .dsk-top-nav {
+    display: flex;
+    list-style: none;
+    gap: 20px;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
```

Comment:
- [ ] The mobile view is not correctly set, you should also add the @media query for the mobile view, and right now the view is pretty bad.

![mb](https://user-images.githubusercontent.com/61562626/135327558-7ea3276d-8db6-4450-bddd-8042cf8e84bf.jpeg)


![mgmm](https://user-images.githubusercontent.com/61562626/135327803-3e8637cc-23fc-4dd5-95ac-a3ce5a3a13f9.png)

Reply:
The navigation is now responsive